tests/logprob/test_binary.py

```diff
@@ -0,0 +1,70 @@
+#   Copyright 2023 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import numpy as np
+import pytensor
+import pytensor.tensor as pt
+import scipy.stats as st
+
+from pymc import logp
+from pymc.testing import assert_no_rvs
+
+
+def test_continuous_rv_comparison_lt():
+    x_rv = pt.random.normal(0.5, 1)
```

Comment:
I would suggest not centering the distribution around the equality point, otherwise both cases will have the same probability and the tests won't notice if they are flipped.

```suggestion
    x_rv = pt.random.normal(0, 1)
```